app/webroot/test.php

```diff
@@ -77,30 +77,30 @@
  *
  */
 if (!defined('WEBROOT_DIR')) {
-	define('WEBROOT_DIR', basename(dirname(__FILE__)));
+    define('WEBROOT_DIR', basename(dirname(__FILE__)));
 }
 if (!defined('WWW_ROOT')) {
-	define('WWW_ROOT', dirname(__FILE__) . DS);
+    define('WWW_ROOT', dirname(__FILE__) . DS);
 }
 
 if (!defined('CAKE_CORE_INCLUDE_PATH')) {
-	if (function_exists('ini_set')) {
-		ini_set('include_path', ROOT . DS . 'lib' . PATH_SEPARATOR . ini_get('include_path'));
-	}
-	if (!include 'Cake' . DS . 'bootstrap.php') {
-		$failed = true;
-	}
+    if (function_exists('ini_set')) {
+        ini_set('include_path', ROOT . DS . 'lib' . PATH_SEPARATOR . ini_get('include_path'));
+    }
+    if (!include 'Cake' . DS . 'bootstrap.php') {
+        $failed = true;
+    }
 } else {
-	if (!include CAKE_CORE_INCLUDE_PATH . DS . 'Cake' . DS . 'bootstrap.php') {
-		$failed = true;
-	}
+    if (!include CAKE_CORE_INCLUDE_PATH . DS . 'Cake' . DS . 'bootstrap.php') {
+        $failed = true;
+    }
 }
 if (!empty($failed)) {
-	trigger_error("CakePHP core could not be found. Check the value of CAKE_CORE_INCLUDE_PATH in APP/webroot/test.php. It should point to the directory containing your " . DS . "cake core directory and your " . DS . "vendors root directory.", E_USER_ERROR);
+    trigger_error("CakePHP core could not be found. Check the value of CAKE_CORE_INCLUDE_PATH in APP/webroot/test.php. It should point to the directory containing your " . DS . "cake core directory and your " . DS . "vendors root directory.", E_USER_ERROR);
```

Comment:
__[staging][code_sniffer]__
* Concat operator must not be surrounded by spaces
* Concat operator must not be surrounded by spaces
* Concat operator must not be surrounded by spaces
* Concat operator must not be surrounded by spaces